pkg/wekafs/mounter.go

```diff
@@ -139,11 +183,11 @@ func (m *wekaMounter) initFsMountObject(fs fsRequest) {
 
 type UnmountFunc func()
 
-func (m *wekaMounter) mountParams(fs string, xattr bool) (string, error, UnmountFunc) {
-	request := fsRequest{fs, xattr}
+func (m *wekaMounter) mountParams(fs string, xattr bool, apiClient *apiclient.ApiClient) (string, error, UnmountFunc) {
```

Comment:
nitpick, but if apiClient can't be part of wekaMounter itself, then its better to have db references, API clients and so on as earlier arguments, and not last one

Reply:
since this param is "optional", put it last

Reply:
but it is not optional
`mounter.incRef(apiClient)` you use it here directly, there is no logic for null

Reply:
ok, found that it's optional much much deeper
i feel that additional incapsulation is missing here
too much of not clear passing of pointers.. but ok, another time